stanford/roles/deploy_manually/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+DEPLOY_MANUALLY_PATH_CONFIG: /opt/configuration
```

Comment:
`DEPLOY_MANUALLY` is the most literal interpretation of this role and why it exists,
which is to say, to run the deploy scripts manually.